src/Models/Argument/ArgumentDefinitions.cs

```diff
@@ -431,6 +432,11 @@ public static class KeyVault
             KeyTypeParam,
             "The type of key to create (RSA, EC).",
             required: true);
+
+        public static readonly ArgumentDefinition<bool> IncludeManagedKeys = new(
+            IncludeManagedKeysParam,
+            "Whether or not to include managed keys in results.",
```

Comment:
Would it be helpful to specify what the default behavior is?

Reply:
The command line library is smart enough to figure it out:
`Whether or not to include managed keys in results. [default: False]`